### ISSUE TITLE
Add PDF export of style guides

### ISSUE THREAD

**Issue by joshsmith** [code-corps/design#1]
We should create a PDF export of the style guides to share with developers who want to create new user interfaces with our designs.

**Comment by ivy-g** [comment]
@JoshSmith are you just asking for a pdf exported from the sketch files that we already have?

**Comment by joshsmith** [comment]
@ivy-g yeah, I think that's a good start on this!

**Comment by ivy-g** [comment]
[Code Corps Style Guide.pdf](https://github.com/code-corps/design/files/610261/Code.Corps.Style.Guide.pdf)
@JoshSmith

**Comment by samserif** [comment]
@ivy-g does a non-pdf version of this exist somewhere? I'm trying to get access to the logo and other UI elements to use for the email templates.

**Comment by joshsmith** [comment]
@samserif yes I have Sketch files. Ping me on Slack and I can give you some of those!
